config/routes.rb

```diff
@@ -472,4 +472,6 @@ def redirect(path, options = { prefix: Whitehall.router_prefix })
   mount Flipflop::Engine => "/flipflop"
 
   mount GovukPublishingComponents::Engine, at: "/component-guide" unless Rails.env.production?
+
+  mount ContentObjectStore::Engine, at: "government/admin/content-object-store"
```

Comment:
curious as I don't know the answer - could we use `admin_root_path` as a variable here rather than hard coding `government/admin`?

Reply:
I'm not aware of a way to do this. Given `admin_root_path` is an output of the configuration earlier in this file I'm not sure it'd be available to us. [I did do a quick read and couldn't see a way to make this dynamic](https://api.rubyonrails.org/v7.1.3.4/classes/ActionDispatch/Routing/Mapper/Base.html#method-i-mount).